// react-native-calendars/src/calendar/day/marking/style.js
import { StyleSheet } from 'react-native';
import * as defaultStyle from '../../style';

import colors from '../../../../../../src/assets/colors/colors'
export default function styleConstructor(theme = {}) {
    const appStyle = { ...defaultStyle, ...theme };
    return StyleSheet.create({
        dots: {
            flexDirection: 'row'
        },
        periods: {
            alignSelf: 'stretch'
        },
        period: {
            height: 4,
            marginVertical: 4, 
            backgroundColor: colors.primary_100,
            height: 22,
        },
        startingDay: {
            marginVertical: 4, 
            paddingHorizontal: 4, 
            backgroundColor: colors.primary_100, 
            borderTopLeftRadius: 2, 
            borderBottomLeftRadius:2,
            marginLeft: 4,
            height: 22,
        },
        endingDay: {
            marginVertical: 4, 
            paddingHorizontal: 4,
            backgroundColor: colors.primary_100, 
            borderTopRightRadius: 2, 
            borderBottomRightRadius:2,
            marginRight: 4,
            height: 22,
            backgroundColor: colors.primary_100
        },
        ...(theme['stylesheet.marking'] || {})
    });
}
